website/views/pages/pricing.ejs

```diff
@@ -11,188 +11,152 @@
-          <div purpose="pricing-tier-card" class="card mr-md-0 mb-4 w-100 w-md-auto">
+          <a purpose="card-button" class="btn btn-block btn-lg btn-primary mx-auto" href="/try-fleet">Get started free</a>
+          <p class="mb-4 mt-4 pt-1"><strong>Includes:</strong></p>
+
+          <%// IT features (free) %>
+          <div purpose="features-list" v-if="!displaySecurityPricingMode">
+            <p>Opt-in macOS MDM</p>
+            <p>REST API and webhooks</p>
+            <p>YAML configuration</p>
+            <p>Enforce device configurations</p>
+            <p>Run MDM commands via CLI</p>
+            <p>End user transparency</p>
+            <p>Community support (MacAdmins Slack)</p>
+            <p>Self-hosted</p>
+          </div>
+
+          <%// Security features (free) %>
+          <div purpose="features-list" v-else>
+            <p>Manage osquery at enterprise scale</p>
+            <p>Monitor query performance</p>
+            <p>REST API and webhooks</p>
+            <p>YAML configuration</p>
+            <p>Detect vulnerabilities</p>
+            <p>Software inventory</p>
+            <p>Device health report</p>
+            <p>Ship logs to Splunk, Snowflake, and more</p>
+            <p>Self-hosted</p>
+          </div>
+        </div>
+        <div purpose="premium-and-cloud-cards" class="d-flex flex-column">
+
+          <%// Fleet Premium tier card %>
+          <div purpose="premium-tier-card" class="card d-flex flex-column">
             <div purpose="pricing-tier-title">
-              <h2>Fleet Ultimate</h2>
-              <h3>For orgs with large deployments</h3>
-              <p>Over 500 hosts<a href="#additional-hosts">†</a></p>
-              <h2>$<span :class="[estimatedUltimateCostPerHostHasBeenUpdated ? 'show-estimated-price' : '']">{{ estimatedUltimateCostPerHost.toFixed(2) }}</span>/host/month</h2>
+              <div class="d-flex flex-sm-row flex-column justify-content-between">
+                <div class="d-flex flex-column justify-content-start">
+                  <h2>Premium</h2>
+                  <h3>Unlimited hosts</h3>
+                </div>
+                <div class="d-flex flex-row align-items-center">
+                  <div class="d-flex flex-row align-items-center ">
+                    <h4>Starting at </h4><h1 class="mb-2">$7.00</h1><strong>/ host <br purpose="premium-price-linebreak">/ month</strong>
+                  </div>
+                </div>
+              </div>
             </div>
             <div>
-              <p><strong>All of Premium plus</strong></p>
-              <%// Ultimate security features %>
-              <div purpose="features-list" v-if="displaySecurityPricingMode">
-                <p>Platinum support</p>
-                <p>Granular role-based access</p>
-                <p>Self-hosted or managed-cloud</p>
+              <a purpose="card-button" class="btn btn-block btn-lg btn-primary mx-auto mb-0" href="/customers/dashboard">Get started</a>
```

Comment:
```suggestion
              <a purpose="card-button" class="btn btn-block btn-lg btn-primary mx-auto mb-0" href="/customers/register">Get started</a>
```

Reply:
Let's point users to `customers/register` rather than `/customers/dashboard`. I think it's safe to assume that users clicking this CTA have not already registered, and if they have, they can click on "I have an account" from the registration page. 

For existing users, this CTA is not an intuitive place for them to log back into their account.